src/effectmanager.cpp

```diff
@@ -392,39 +392,29 @@ void EffectManager::ApplyGlobalPaletteColors()
     auto& deviceConfig = g_ptrSystem->DeviceConfig();
     auto& globalColor = deviceConfig.GlobalColor();
     auto& secondColor = deviceConfig.SecondColor();
-
     #if (USE_HUB75)
-            auto pMatrix = g();
+        auto pMatrix = g();
 
-            // If the two colors are the same, we just shift the palette by 64 degrees to create a palette
-            // based from where those colors sit on the spectrum
-            if (secondColor == globalColor)
-            {
-                CHSV hsv = rgb2hsv_approximate(globalColor);
-                pMatrix->setPalette(CRGBPalette16(globalColor, CRGB(CHSV(hsv.hue + 64, 255, 255))));
-            }
-            else
-            {
-                // But if we have two different colors, we create a palettte spread between them
-                pMatrix->setPalette(CRGBPalette16(secondColor, globalColor));
-            }
-            pMatrix->PausePalette(true);
+        // If the two colors are the same, we just shift the palette by 64 degrees to create a palette
+        // based from where those colors sit on the spectrum
+        if (secondColor == globalColor)
+        {
+            CHSV hsv = rgb2hsv_approximate(globalColor);
+            pMatrix->setPalette(CRGBPalette16(globalColor, CRGB(CHSV(hsv.hue + 64, 255, 255))));
+        }
+        else
+        {
+            // But if we have two different colors, we create a palettte spread between them
+            pMatrix->setPalette(CRGBPalette16(secondColor, globalColor));
+        }
+        pMatrix->PausePalette(true);

```

Comment:
I understand why you're doing this, but if you change nothing in the code of a block then please don't modify the white spacing either.

Reply:
This was an artifact that occurred when I was deciding how to implement the change I wanted. I originally wrapped that section in an if else statement. But, I quickly realized that was a stupid idea and removed the conditional. Now that I am looking at the original upstream file, I notice there is an extra indent in that section. The block in the if section does not match the indent in the else section. Since I am already proposing change to the file is it okay to also "fix" the indent?

Reply:
No, please don't. The block itself is consistently indented, so it's indented to deep but not confusing. This may not make much sense to you, but I have to review a **lot** of PRs, so reducing whitespace noise really saves me a lot of energy - and sanity.

Reply:
Does your review tool not have a configuration to ignore whitespace?

Reply:
That's not the point. Whitespace _does_ matter, and changes to it _should_ be an improvement. We just don't merge big blocks of whitespace-only changes, and they do distract from other stuff.

Reply:
At this point, I don't know what the spacing was. But, the last few commits have made the spacing what it should be to conform to the spacing standards of the project. Unless I am missing something obvious.